### ISSUE TITLE
Format removes '=' and breaks code in some circumstances

### ISSUE THREAD

**Issue by Phil-Pinkowski** [pinterest/ktlint#1103]
## Expected Behavior

When I run the formatter on the following code snippet it should not produce invalid/broken code:

```kotlin
fun test(input: String): String
 = input
```

## Observed Behavior

Running the formatter with the above code produces the following broken code:

```kotlin
fun test(input: String): String
input
```

## Steps to Reproduce

Run the formatter on a file with the content specified above, e.g. `ktlint -F example.kt`

## Your Environment

* Version of ktlint used: 0.40.0
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): command line version 0.40.0
* Version of Gradle used (if applicable):
* Operating System and version: Mac OS Big Sur (11.0.1)

**Comment by paul-dingemans** [comment]
I can reproduce the problem with ktlint version "0.40.0". With current development version the problem does not occur.

**Comment by Tapchicoma** [comment]
It was fixed in upcoming `0.41.0` release. Please give it a try once it will be published.
